src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5455,29 +5455,38 @@ private string CreateManifestString()
 
                     // write out each enum value 
                     FieldInfo[] staticFields = enumType.GetFields(BindingFlags.DeclaredOnly | BindingFlags.Public | BindingFlags.Static);
+                    bool anyValuesWritten = false;
```

Comment:
This fix allows enums with unsigned underlying types.  It also insures that if there are no enum values we add one because the windows ETW manifest parser requires it.